appium/api-mocks/core/msg-block-parser.ts

```diff
@@ -101,7 +52,8 @@ export class MsgBlockParser {
   }
 
   public static stripPublicKeys = (decryptedContent: string, foundPublicKeys: string[]) => {
-    const { blocks, normalized } = MsgBlockParser.detectBlocks(decryptedContent);
+    // eslint-disable-next-line prefer-const
+    let { blocks, normalized } = MsgBlockParser.detectBlocks(decryptedContent);
```

Comment:
Maybe we can use like this? I think it's good to avoid `disable eslint rules` if possible
```ts
const { blocks, normalized } = MsgBlockParser.detectBlocks(decryptedContent);
let changedNormalized = normalized;
```